.github/workflows/links.yml

```
@@ -21,19 +21,23 @@ jobs:
           args: "--base . --cache --max-cache-age 1d . --max-redirects 10 --max-retries 5 --user-agent Chrome/51.0.2704.103 Safari/537.36"
 
+      - name: Set lychee exit code
+        run: echo "lychee_exit_code=$?" >> $GITHUB_ENV
+
       - name: Save lychee cache
         uses: actions/cache/save@v4
         if: always()
         with:
           path: .lycheecache
-          key: ${{ steps.restore-cache.outputs.cache-primary-key }}
+          key: ${{ steps.restore-cache.outputs.cache-key }}
```

actionlint (1.7.4)

property "cache-key" is not defined in object type {cache-hit: string; cache-matched-key: string; cache-primary-key: string}

(expression)